error.vue

```diff
@@ -39,20 +30,22 @@ async function reload() {
         <span timeline-title-style>Error</span>
       </template>
       <slot>
-        <form p5 grid gap-y-4 @submit="reload">
-          <div text-lg>
-            Something went wrong
+        <div p5 grid gap-y-4>
+          <div text-xl font-bold>
+            {{ message }}
           </div>
           <div text-secondary>
-            {{ message }}
+            {{ subtitle }}
           </div>
-          <button flex items-center gap-2 justify-center btn-solid text-center :disabled="state === 'reloading'">
-            <span v-if="state === 'reloading'" block animate-spin preserve-3d>
-              <span block i-ri:loader-2-fill />
-            </span>
-            {{ state === 'reloading' ? 'Reloading' : 'Reload' }}
-          </button>
-        </form>
+          <NuxtLink
+            v-if="showBack"
+            :aria-label="$t('nav.back')"
+            class="btn-text inline-flex flex-items-center -ml-5"
+            @click="$router.go(-1)"
```

Comment:
One future improvement here is to make sure if they landed directly onto this error page due to poorly formed url or something that we send them to the base experience.  Again, my understanding of how the nuxt router is limited, but if it just leverages the history api, this might mean `go back to the previous page, which may be another site` ... 

Not a blocker, but maybe something we can confirm and possible slate for a follow up.

Reply:
If it is a counterpart to the `Next` router, then this is my suspicion of what happens too

Reply:
I'll be sure to test when it goes to staging